### PR TITLE
Translation improvements

### PR DESCRIPTION
**Pull request by stejskalleos**
* Extract missing strings for the translations
* Update config for the transifex-cli
* Update *.po files
* Fix interpolations in strings

### CODE REVIEW

**Review by ekohl — APPROVED** [review]
It should be noted that right now no `.mo` are present. We can solve this in packaging, but don't do that today. That's still something to resolve to make the translations usable in practice.

**Comment by stejskalleos** [comment]
> It should be noted that right now no `.mo` are present. We can solve this in packaging, but don't do that today. That's still something to resolve to make the translations usable in practice.

Should I add them?

File: locale/cs/hammer_cli_foreman_ansible.po [hunk 1]
```diff
@@ -0,0 +1,449 @@
+# Czech translations for hammer_cli_foreman_ansible package.
```
Comment (ekohl) [inline locale/cs/hammer_cli_foreman_ansible.po:1]: You're adding this file, but it doesn't appear to include any translations. We normally only do this with > 50% translations.

**Review by ekohl — CHANGES_REQUESTED** [review]
I just realized all the translation files are empty. I'd leave them out until we have translations.

**Comment by ofedoren** [comment]
Closing in favor of https://github.com/theforeman/hammer-cli-foreman-ansible/pull/21

**Pull request closed without merging**